setup.py

```diff
@@ -0,0 +1,46 @@
+from setuptools import setup, find_packages
+# To use consistent encoding
+from codecs import open
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+# Fetch the long description form the separate README.rst file
+with open(path.join(here, 'README.rst'), encoding = 'utf-8') as f:
+    long_description = f.read()
+
+version = {}
+with open(path.join(here, 'VERSION'), encoding = 'utf-8') as version_file:
+    version = version_file.read().strip()
+    
+setup(
+    name = 'UnsplashUbuntu',
+    version = version,
+
+    description = 'Wallpaper Changer with multi-monitor support',
+    long_description = long_description,
+
+    url = 'https://github.com/PseudoAj/UnsplashUbuntu',
+
+    author = 'PseudoAj',
+    author_email = '',
+
+    license = 'GPL v3',
```

Comment:
Not sure if there are any reservations; Using and persuading to give back in an open source fashion feels fine